### ISSUE TITLE
After moving a Section that has Columns in it, you can't resize the Columns via the "Column Width" field

### ISSUE THREAD

**Issue by Pikamander2** [pojome/elementor#5393]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

## Description

Normally I can resize a column by changing the "Column Width" field.

But sometimes the column width will refuse to apply/save the new value and will throw this error:

```
Uncaught TypeError: Cannot read property 'errors' of undefined
    at n.onEditorInlineSizeInputChange (editor.min.js?ver=2.1.6:2)
    at i.function.l.(/wp-admin/anonymous function) [as validationMethod] (https://example.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.1.6:2:217231)
    at isValid (editor.min.js?ver=2.1.6:2)
    at i.function.l.(/wp-admin/anonymous function) [as isValid] (https://example.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.1.6:2:217231)
    at editor.min.js?ver=2.1.6:2
    at Array.every (<anonymous>)
    at n.onBaseInputChange (editor.min.js?ver=2.1.6:2)
    at HTMLDivElement.dispatch (load-scripts.php?c=0&load[]=underscore,shortcode,utils,jquery-core,jquery-migrate,backbone,wp-util,wp-backbone,media-models,moxiejs,plupload,wp-plupload,jqu&load[]=ery-ui-core,jquery-ui-widget,jquery-ui-mouse,jquery-ui-sortable&ver=4.9.8:11)
    at HTMLDivElement.r.handle (load-scripts.php?c=0&load[]=underscore,shortcode,utils,jquery-core,jquery-migrate,backbone,wp-util,wp-backbone,media-models,moxiejs,plupload,wp-plupload,jqu&load[]=ery-ui-core,jquery-ui-widget,jquery-ui-mouse,jquery-ui-sortable&ver=4.9.8:11)
```

It only happens from time to time, so it was difficult to figure out what the trigger is. 

It turns out that it's caused by moving a Section (row) that has a Column widget inside of it.

## Steps to reproduce

1. Create a page that uses Elementor
2. Add a Section to the page.
3. Add another Section to the page that contains the Columns widget.
4. Move the Column widget's Section to another location (you have to move it somewhere different; it can't remain in the same spot).
5. Click Edit Column on one of the columns
6. Attempt to change its Column Width under the Layout tab.
7. The column will not change in size, nor will it save your new value. Additionally, the console will produce that error.

Here's a video summarizing those steps and demonstrating the error:
[Streamable link](https://streamable.com/khsaq)
[Zipped MP4 file](https://github.com/pojome/elementor/files/2281645/Elementor.column.bug.zip)

This issue has occurred on two very different environments, one of which was almost a fresh install of WordPress but had Elementor installed.

Updating and refreshing the page will fix the issue, so my guess is that some kind of column reference isn't being copied when the Section is moved. Interestingly, you can still manually drag the column divider to change their widths.

## Isolating the problem
- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a default WordPress theme active.
- [X] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: 10.1.31
	PHP Version: 7.0.31
	PHP Max Input Vars: 1000
	PHP Max Post Size: 32M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 4.9.8
	Site URL: (any site)
	Home URL: (any site)
	WP Multisite: No
	Max Upload Size: 32 MB
	Memory limit: 40M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: America/New_York
	Debug Mode: Inactive

== Theme ==
	Name: Twenty Seventeen
	Version: 1.7
	Author: the WordPress team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.106 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 2.1.6
		Author: Elementor.com


== Debug ==
	Errors: 
		TypeError: Cannot read property 'errors' of undefined
			at http://example.com/tempbugtest1/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.1.6 - 2:139163
			2018-08-13 08:53 +00:00
			x 23 times

				
```

**Comment by bainternet** [comment]
@Pikamander2 

We can't replicate, 
Please contact our support at support @elementor.com for guidance in resolving this issue and mention that you were directed by me and reference this issue.

thanks.

**Comment by Pikamander2** [comment]
@bainternet - I'll get in touch with them, though the issue seems to occur pretty consistently. Are you positive that you tried the steps in the same order as the video? Do you think that you could record yourself going through the steps so that I could take a look? I'd really like for this to get fixed, as I'm sure that other users have run into it as well.

**Comment by arielk** [comment]
This issue is resolved in the beta version of Elementor v2.2:
https://github.com/pojome/elementor/issues/5493

Thanks for the feedback!
